src/types/redis_json.h

```diff
@@ -82,6 +82,7 @@ class Json : public Database {
 
   rocksdb::Status Resp(engine::Context &ctx, const std::string &user_key, const std::string &path,
                        std::vector<std::string> *results, RESP resp);
+  static rocksdb::Status FromString(const std::string &value, JsonValue *result);
```

Comment:
If you really need it, please rename it to `FromRawString`. `FromString` is a weird name here.

Reply:
Yes, `FromRawString` would be better.